GraphicEditor/ViewModel/CardiologyBuildingUserControlViewModel.cs

```diff
@@ -0,0 +1,95 @@
+﻿using GraphicEditor.HelpClasses;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicEditor.ViewModel
+{
+    public class CardiologyBuildingUserControlViewModel : BindableBase
+    {
+        private MapContentUserControlViewModel _parent;
+        string[] array = { "Basement", "Ground Floor", "First Floor", "Second Floor", "Third Floor" };
+        private List<string> _floors;
+        private string _selectedFloor;
+        public MyICommand<string> NavCommand { get; private set; }
+        public static CardiologyFirstFloorMapUserControlViewModel FirstFloor;
+        public static CardiologySecondFloorMapUserControlViewModel SecondFloor;
+        public BindableBase _floorViewModel;
+
+
+
+        public CardiologyBuildingUserControlViewModel(MapContentUserControlViewModel parent)
+        {
+            _parent = parent;
+            _floors = new List<string>(array);
+            _selectedFloor = _floors[2];
```

Comment:
It's not recomended to use numbers as indexes. Use constants that represent numbers, or in this case, it would be best if you used constants.FIRS_FLORE instead of _floors[2]

Reply:
I think you are wrong here. I really don't need a constant for this because this happens only once.

Reply:
Nope, this is just a good clean code practise. It doest have anything to do with number of times you use that constant.

Reply:
No, @VladimirBudjen is definitely right here, even if it is used only here, although it is not guaranteed that it will be in the future, it is extremely unreadable to leave a random index number. What difference does it make if you write 0 or 4 or whatever other number?

Reply:
Alright, we will introduce these constants into the Constants.cs

Reply:
You don't need to put the default floor number in the Constants class, you could just use a constant field in this one